modules/mobile/app/models/mobile/v0/adapters/vaos_v2_appointments.rb

```diff
@@ -133,7 +155,10 @@ def cancel_id(appointment_hash)
         end
 
         def type_of_care(service_type, type)
-          va?(type) ? nil : service_type
+          return nil unless service_type
```

Comment:
If type is VA, we return nil, so it would probably make sense to do something like:
```
return nil if service_type.nil? || va?(type)
```

Reply:
good catch. done